src/Http/Session.php

```diff
@@ -217,9 +217,9 @@ public function regenerateId()
 				throw new Nette\InvalidStateException('Cannot regenerate session ID after HTTP headers have been sent' . ($file ? " (output started at $file:$line)." : '.'));
 			}
 			if (session_id() !== '') {
+				session_write_close();
```

Comment:
Maybe the `session_write_close();` should be before the if. That should fix the failing test as well.